third_party/blink/renderer/core/farbling/brave_session_cache.h

```diff
@@ -41,7 +42,8 @@ enum FarbleKey : uint64_t {
 };
 
 typedef absl::randen_engine<uint64_t> FarblingPRNG;
-typedef base::RepeatingCallback<float(float, size_t)> AudioFarblingCallback;
+typedef absl::optional<base::RepeatingCallback<float(float, size_t)>>
```

Comment:
i'd rather call it `OptionalAudioFarblingCallback` or explicitly use optional everywhere, b/c now it is not clear from the type name that the callback can be empty

Reply:
Good idea. Changed in https://github.com/brave/brave-core/pull/15894/commits/5dfe6fd19a42ac3f0f61f2c7a2d490ecca4448dd